### PR TITLE
B3 Case Importer from Excel

### PR DESCRIPTION
**Pull request by biyeun**
see screenshots

screen shot 2016-02-17 at 5 20 32 pm
screen shot 2016-02-17 at 5 46 03 pm
screen shot 2016-02-17 at 5 55 34 pm
screen shot 2016-02-17 at 6 04 46 pm

### CODE REVIEW

**Comment by biyeun** [comment]
merge and rebase: https://github.com/dimagi/commcare-hq/pull/10349

File: corehq/apps/importer/base.py [hunk 1]
```diff
@@ -9,3 +9,6 @@ class ImportCases(DataInterface):
     report_template_path = "importer/import_cases.html"
     gide_filters = True
     asynchronous = False
+
+    is_bootstrap3 = True
+
```
Comment (dimagimon) [inline corehq/apps/importer/base.py]: W391 blank line at end of file

**Comment by biyeun** [comment]
QA ticket submitted and feature is on staging

**Comment by millerdev** [comment]
Reviewed quickly

File: corehq/apps/importer/templates/importer/excel_config.html [hunk 2]
```diff
@@ -1,15 +1,10 @@
...
 
-{% block head %}
-    {{ block.super }}
-{% endblock %}
-
-{% block js %}
-    {{ block.super }}
-    <script src="{% static 'style/js/bootstrap2/main.js' %}"></script>
+{% block js %}{{ block.super }}
+    <script src="{% new_static 'style/js/bootstrap2/main.js' %}"></script>
```
Comment (emord) [inline corehq/apps/importer/templates/importer/excel_config.html]: @biyeun merged with master this was `hqwebapp/js/main.js` which according to https://github.com/dimagi/commcare-hq/pull/10502/files translates to `bootstrap2/main.js` I'm assuming it should actually be moved to bootstrap3?

Comment (biyeun) [inline corehq/apps/importer/templates/importer/excel_config.html]: yep, that's correct! Thanks for the note. Will take care of this now...

**Comment by biyeun** [comment]
qa complete cc @proteusvacuum

**Comment by biyeun** [comment]
also cc @millerdev since you were looking

File: corehq/apps/importer/templates/importer/partials/import_status.html [hunk 3]
```diff
@@ -80,8 +80,12 @@
         </p>
     {% endif %}
     {% if progress.percent %}
-        <div class="progress progress-striped active">
-            <div class="bar" style="width: {{ progress.percent }}%;"></div>
+        <div class="progress">
+            <div class="progress-bar progress-bar-info"
+                 role="progressbar" aria-valuenow="60" aria-valuemin="0" aria-valuemax="100"
```
Comment (proteusvacuum) [inline corehq/apps/importer/templates/importer/partials/import_status.html]: should `aria-valuenow` be `{{progress.percent}}`? 
Didn't realize we did accessibility stuff... if this isn't really something we are supporting, might be worth just removing the `aria` values all together?

Comment (biyeun) [inline corehq/apps/importer/templates/importer/partials/import_status.html]: it also doesn't hurt to have them in, and we have those present in a number of places.

Comment (proteusvacuum) [inline corehq/apps/importer/templates/importer/partials/import_status.html]: fair enough. still think it should be `{{progress.percent}}` though.

**Pull request merged**